activities.py

```diff
@@ -0,0 +1,62 @@
+# @@@SNIPSTART python-money-transfer-project-template-withdraw
+from temporalio import activity
+
+from shared import PaymentDetails
+from banking_service import BankingService, InvalidAccountError, InsufficientFundsError
+
+# Generic errors for activity failures
+class WithdrawFailedError(Exception):
+    pass
+
+class DepositFailedError(Exception):
+    pass
+
+class RefundFailedError(Exception):
+    pass
+
+@activity.defn
+async def withdraw(data: PaymentDetails) -> str:
+    reference_id: str = f"{data.reference_id}-withdrawal"
+    bank: BankingService = BankingService("bank-api.example.com")
```

Comment:
Can consider demonstrating activity methods on a class with banking service shared across them. See https://github.com/temporalio/samples-python/blob/main/hello/hello_activity_method.py

Reply:
For a future tutorial.